clients/cobol-lsp-vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
+Changelog:
+
+## [0.19.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.18.0...0.19.0) (2021-06-08)
+
+#### Added
+* IDMS DML commands support
```

Comment:
this is listed in both 0.19.0 and 0.18.0. Should it be "expanded support for IDMS DML commands"? as in 0.19.0 supports more of them?

Reply:
Yeah, makes sense